near/omni-bridge/src/btc.rs

```diff
@@ -171,4 +168,22 @@ impl Contract {
             PromiseOrValue::Value(())
         }
     }
+
+    #[access_control_any(roles(Role::DAO))]
+    pub fn add_btc_connector(&mut self, chain_kind: ChainKind, btc_connector_id: AccountId) {
```

Comment:
I'd prefer to use `utxo` in the names

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/3ef0d762115ebde273c4694ae7e92c968586b368